packages/react/src/components/ProgressBar/ProgressBar.jsx

```diff
@@ -98,13 +102,74 @@ const ProgressBar = ({
   const matchingFillColor = matchingThreshold?.color ?? DEFAULT_PROGRESS_BAR_COLOR;
   const fillColor = hasColorObject ? matchingThreshold.color.fill : matchingFillColor;
   const strokeColor = hasColorObject ? matchingThreshold.color.stroke : undefined;
+  const valueAndUnit = Number.isInteger(value) ? `${value}${valueUnit}` : '';
+
+  /**
+   * helper to render the value and unit
+   *
+   * @returns JSXElement
+   */
+  const renderValueAndUnit = () => (
```

Comment:
Why do we call these as functions rather than use the JSX?

Reply:
Just habit to create a render function. No reason it can't be jsx in this case. Updated and pushed.